wikimedia/wikipedia-ios
# [ALT TEXT] Show survey component

**Phabricator:** https://phabricator.wikimedia.org/T370308

### Notes
This PR reuses the image recommendations survey component to display when tapping "No" on the alt text prompt.  

### Test Steps
1. Fresh install, turn on feature flags as needed.
2. Go through image recommendations flow, after publishing, tap No on the alt text prompt.
3. Confirm survey displays. Confirm tapping cancel brings you back to image recommendation. Repeat, and confirm tapping Submit also brings you back to image recommendation.
4. Edit an article on Test Wiki (I've been using the IOS article). Confirm after making a random edit, an alt text prompt appears. Tap No. Confirm survey displays. Confirm tapping cancel or submit brings you back to the article.
